### PR TITLE
Merging CENM 1.5.6

### PR DESCRIPTION
**Pull request by paulmoloneyr3**

### CODE REVIEW

File: content/en/platform/corda/1.5/cenm/legal-info-1.5.5.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ---
-date: '2022-01-28T09:59:25Z'
+date: '2023-01-20'
 menu:
   cenm-1-5:
-    identifier: cenm-1-5-5-legal-info
-    parent: cenm-1-5-cenm-releases
+    identifier: cenm-1-5-6-legal-info
+    parent: cenm-1-6-cenm-releases
```
Comment (yarivus) [inline content/en/platform/corda/1.5/cenm/legal-info-1.5.5.md:6]: should the parent be cenm-1-6 or cenm-1-5 ?

**Review by yarivus — COMMENTED** [review]
@paulmoloneyr3 see my question/comment

File: content/en/platform/corda/1.5/cenm/legal-info-1.5.5.md (hunk 1 above)
Comment (paulmoloneyr3) [reply content/en/platform/corda/1.5/cenm/legal-info-1.5.5.md:6]: Hi Yariv, fixing this in this PR:

https://github.com/corda/corda-docs-portal/pull/985

**Review by yarivus — APPROVED** [review]
LGTM

**Pull request closed without merging**